tensor_api.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<p>now there are a bunch of new ones, please also update them in the docs: <a href="https://github.com/intel-isl/Open3D/pull/2066#issue-447411752" target="_blank">https://github.com/intel-isl/Open3D/pull/2066#issue-447411752</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/intel-isl/Open3D/pull/2067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>